apply_factors_wrangler.py

```diff
@@ -183,7 +191,7 @@ def lambda_handler(event, context):
             )
         )
 
-        filtered_data = final_imputed.drop(cols_to_drop, axis=1)
+        filtered_data = final_imputed.drop(cols_to_drop, axis=1, errors='ignore')
```

Comment:
Due to our changes to Calculate Factors, the only columns that need dropping are the "imputation_factor_" columns. As such can we refactor the cols_to_drop variable to reflect this and we would no longer need to use, errors='ignore'.

Reply:
quite right, gonna go test this on aws